scalajslib/src/mill/scalajslib/api/ScalaJSApi.scala

```diff
@@ -116,3 +94,44 @@ object JsEnvConfig {
       autoExit: Boolean
   ) extends JsEnvConfig
 }
+
+final class Report private (val publicModules: Iterable[Report.Module]) {
```

Comment:
I know that all API case classes are in this file, but personally, I'd prefer to have a dedicated `Report.scala` file for it.

Reply:
Good idea! I'm going to move it to a separate file.